### ISSUE TITLE
oxlint: space-separated rule list in disable-next-line silently disables nothing

### ISSUE THREAD

**Issue by samuelbriole** [oxc-project/oxc#22294]
## Summary

When using `// oxlint-disable-next-line` with a space-separated (instead of comma-separated) list of rule names, oxlint silently disables **no** rules — every listed rule still reports. This makes the comment look effective when it is not.

## Repro

`oxlint --version`: `1.63.0`

`.oxlintrc.json`:
```json
{
  "rules": {
    "no-unused-vars": "error",
    "no-console": "error"
  }
}
```

`test.js`:
```js
// Case A: comma-separated (works correctly)
// oxlint-disable-next-line no-unused-vars, no-console
function fn1() { console.log("hi"); }

// Case B: comma-separated, no space (works correctly)
// oxlint-disable-next-line no-unused-vars,no-console
function fn2() { console.log("hi"); }

// Case C: space-separated (BUG: nothing is disabled)
// oxlint-disable-next-line no-unused-vars no-console
function fn3() { console.log("hi"); }
```

Run:
```sh
oxlint test.js
```

## Actual

`fn1` and `fn2`: clean (correct).
`fn3`: both `no-unused-vars` and `no-console` still report.

## Expected

Either:
1. Space-separated rule names should be supported (this is what ESLint accepts), OR
2. oxlint should warn / error on the malformed comment so users notice the typo.

The current silent-failure behavior is the worst of both worlds: it's easy to write a comment that looks fine, passes lint, and disables nothing.

## Context

We hit this porting custom rules from ESLint to oxlint in a large monorepo migration. Several pre-existing `eslint-disable-next-line ruleA ruleB` comments were converted to `oxlint-disable-next-line ruleA ruleB` and silently lost their effect. We had to audit every multi-rule disable comment and convert them to comma-separated form.

**Comment by camc314** [comment]
apologies - please use the previous version. this will be fixed in the release on monday
